source/manual/get-ssh-access.html.md

```diff
@@ -21,21 +21,7 @@ User accounts in our integration environments are managed in the [govuk-puppet][
     mac$ cd ~/govuk
     mac$ git clone git@github.com:alphagov/govuk-puppet.git
 
-To create a new account, start by creating an SSH key at least 4096 bits long. For example:
-
-    mac$ ssh-keygen -t rsa -b 4096 -C "your_email@example.com" -f ~/.ssh/alphagov
-
-Import the SSH key into your keychain.
-
-    mac$ /usr/bin/ssh-add -K ~/.ssh/alphagov
-
-You'll also want to add the above line into your `~/.bash_profile` so that it is persistent between restarts.
-    
-Create a public key from this:
-
-    mac$ ssh-keygen -y -f ~/.ssh/alphagov > ~/.ssh/alphagov.pub
-
-Now create a user manifest in `~/govuk/govuk-puppet/modules/users/manifests` with your username and the public key you just created. Your username should use the `firstnamelastname.pp` format.
```

Comment:
Disclaimer: I think I was the one that added these ssh instructions in the first place. I think there's value in keeping them because GitHub doesn't actually show you how to create a public key from your private one, nor does it have the instruction around making the SSH key import persistent between restarts.

I agree it's probably in the wrong place though - perhaps it could be moved to the same file where the GitHub setup page is linked?

Reply:
I just tried this and the public key was auto-generated by the initial command:

```
$ ssh-keygen -t rsa -b 4096 -C "your_email@example.com" -f ~/.ssh/alphagov
[...]
Your identification has been saved in /Users/isabelllong/.ssh/alphagov.
Your public key has been saved in /Users/isabelllong/.ssh/alphagov.pub.
``

Reply:
Done!

Reply:
So it does! Hmmm. Could we refer to the output in the docs, along the lines of "the previous command will generate a private and a public key. You'll need to grab the contents of your public key and save them HERE:".

Reply:
This is already here:

>     ssh_key  => 'this public key will be a few lines long (copy the output from `more ~/.ssh/alphagov.pub`). It should begin with `ssh-rsa AAA` and end with `== john.smith.123@digital.cabinet-office.gov.uk`. You may need to add the email address to the end of your public key manually.',

Line 31.

Reply:
Ah, perfect. Thanks!